docs/spec/overview.md

```diff
@@ -56,12 +56,12 @@ For more details, see [Kind: Subscription](spec.md#kind-subscription).
 
 ## Channel
 
-**Channel** provides an at least once event delivery mechanism which can fan
-out received events to multiple destinations via _Subscriptions_. A _Channel_
-has a single inbound _Sinkable_ interface which may accept events from multiple
-_Subscriptions_ or even direct delivery from external systems. Different
-_Channels_ may implement different degrees of persistence. Event delivery order
-is dependent on the backing implementation of the _Channel_ provided by the
+**Channel** provides an event delivery mechanism which can fan out received
```

Comment:
This relaxes the Channel guarantee to allow at-most-once delivery. This is probably ok but @vaikas-google should probably be the one to approve.

Reply:
FWIW, I tend to think separately about reliability and durability of messages.

In this case, we expect the Channel to be reliable, in that it will retry delivery ("at least once") to each endpoint for each event. For the in-memory channel, we don't necessarily expect the events to be durable during unexpected events (i.e. process/system failure or restart), but for events which are not internally destroyed, we will retry delivery. In the spec, I tried to represent this as:

> provides an at least once event delivery
> ...
> may implement different degrees of persistence _(should have been **durability**)_

With that said, I think it's fine for now to relax the reliability expectations on Channel.